### PR TITLE
[RM Ch3] Types of Accelerators

### PR DESCRIPTION
**Pull request by petorre**
Towards fixing #2051 .

### CODE REVIEW

File: doc/ref_model/chapters/chapter03.md [hunk 1]
```diff
@@ -282,7 +282,22 @@ The terms Container Infrastructure Service Instance and Container Infrastructure
...
 
 <a name="3.8.1"></a>
-### 3.8.1 Types of accelerators
+### 3.8.1 Types of Accelerators
+
+Accelerator technologies can be categorized depending on where they are realized in the hardware product and how they get activated, life cycle managed and supported in running infrastructure.
+
+| Acceleration technology/hardware | Example implementation | Activation/LCM/support | Usage by application tenant |
+|---|---|---|---|
+| CPU instructions | Within CPU cores | None for hardware | Could be application binary differences (application to load software library that recognizes and uses) |
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter03.md:291]: "Could be application binary differences" would refer to with and without the additional CPU instructions. The column title, however, refers to "Usage". Should this just state what is currently in the parentheses and the outcome "application binary differences between versions that use and do not use these instructions" can follow.

File: doc/ref_model/chapters/chapter03.md [hunk 2]
```diff
@@ -282,7 +282,22 @@ The terms Container Infrastructure Service Instance and Container Infrastructure
...
+| Firmware-programmable adapter | Network/storage adapter with programmable part of firmware image | Rare updates | Application normally not modified or aware |
+| SmartNIC | Programmable accelerator for vSwitch/vRouter, NF and/or Hardware Infrastructure | Programmable by Infrastructure operator(s) and/or application tenant(s) | 3 types/operational modes: 1. Non-programmable normally with unaware applications; 2. Once programmable to activate; 3 Reprogrammable |
+| SmartSwitch-based | Programmable Switch Fabric or TOR switch | Programmable by Infrastructure operator(s) and/or application tenant(s) | 3 operational modes: 1. Non-programmable normally with unaware applications; 2. Once programmable to activate; 3. Reprogrammable |
+
+<p align="center"><img src="../figures/ch03-example-of-server-and-smartswitch-based-nodes.png" alt="Examples of server- and SmartSwitch-based nodes (for illustration only)" Title="Examples of server- and SmartSwitch-based nodes (for illustration only)" width="65%"/></p>
+<p align="center"><b>Figure 3-7:</b> Examples of server- and SmartSwitch-based nodes (for illustration only)</p>
```
Comment (CsatariGergely) [inline doc/ref_model/chapters/chapter03.md:298]: Figure added in #2160

Comment (petorre) [reply doc/ref_model/chapters/chapter03.md:298]: > Figure added in #2160

Thanks

**Review by pgoyal01 — APPROVED** [review]

File: doc/ref_model/chapters/chapter03.md (hunk 1 above)
Comment (petorre) [reply doc/ref_model/chapters/chapter03.md]: > 
> 
> "Could be application binary differences" would refer to with and without the additional CPU instructions. The column title, however, refers to "Usage". Should this just state what is currently in the parentheses and the outcome "application binary differences between versions that use and do not use these instructions" can follow.

Yes, changed. Thanks

**Review by walterkozlowski — APPROVED** [review]

**Comment by walterkozlowski** [comment]
@TFredberg, @qabpea can you please review and approve if you can?

**Review by TFredberg — APPROVED** [review]

**Pull request merged**